website/docs/concepts/router.md

```diff
@@ -3,13 +3,20 @@ title: "Router"
 description: "Yew's official router"
 ---
 
-[The router on crates.io](https://crates.io/crates/yew-router)
```

Comment:
This is exclusive for `yew/next` and crates.io link will point people to published old documentation of yew_router. A caveat is we need to change the.